sklearn/datasets/base.py

```diff
@@ -440,6 +446,10 @@ def load_digits(n_class=10):
         sample, 'target_names', the meaning of the labels, and 'DESCR',
         the full description of the dataset.
 
+    (data, target) : tuple if ``return_X_y`` is True
+
+    .. versionadded:: 0.18
```

Comment:
This is what it currently looks like
![screenshot from 2016-08-06 16-59-14](https://cloud.githubusercontent.com/assets/12952080/17456504/35d8d996-5bf7-11e6-8afb-758f2e89bf56.png)